### PR TITLE
tests: Bluetooth: Tester: Format BTP cmd/evt/rsp logs as hex

### PR DESCRIPTION
**Pull request by Thalley**
Since the values are defined as hex, e.g. 0x82, it is easier to compare with the log if they also log them as such.

### CODE REVIEW

**Review by jhedberg — APPROVED** [review]

File: tests/bluetooth/tester/src/btp.c [hunk 1]
```diff
@@ -96,8 +96,8 @@ static void cmd_handler(void *p1, void *p2, void *p3)
 
 		cmd = k_fifo_get(&cmds_queue, K_FOREVER);
 
-		LOG_DBG("cmd service %u opcode %u index %u", cmd->hdr.service, cmd->hdr.opcode,
-			cmd->hdr.index);
+		LOG_DBG("cmd service 0x%02X opcode 0x%02X index 0x%02X", cmd->hdr.service,
```
Comment (jhedberg) [inline tests/bluetooth/tester/src/btp.c:99]: The BTP spec uses lower-case hex, so seems like `0x%02x` would be more appropriate. I believe it's the common form in most of the Bluetooth code base too.

**Review by jhedberg — CHANGES_REQUESTED** [review]

**Review by jhedberg — DISMISSED** [review]

File: tests/bluetooth/tester/src/btp.c [hunk 2]
```diff
@@ -96,8 +96,8 @@ static void cmd_handler(void *p1, void *p2, void *p3)
 
 		cmd = k_fifo_get(&cmds_queue, K_FOREVER);
 
-		LOG_DBG("cmd service %u opcode %u index %u", cmd->hdr.service, cmd->hdr.opcode,
-			cmd->hdr.index);
+		LOG_DBG("cmd service 0x%02x opcode 0x%02x index 0x%02x", cmd->hdr.service,
```
Comment (sjanc) [inline tests/bluetooth/tester/src/btp.c:99]: services are defined as decimal

Comment (Thalley) [reply tests/bluetooth/tester/src/btp.c:99]: :upside_down_face:

**Comment by Thalley** [comment]
@sjanc 
In Zephyr the services are defined as decimal, opcodes and error codes are hex, indexes I'm not sure

In the autopts client everything seems to be logged as decimal, e.g. `Header(svc_id=0, op=128, ctrl_index=255, data_len=0)`, but where services are (luckily) also decimal and opcodes are hex. 

We really should align these so that it's possible to compare not only the definitions, but also the logs. 

Is there a reason why the services are decimal?

**Review by sjanc — DISMISSED** [review]
no reason (other than it is defined like this in autopts docs),  we should just switch to hex everywhere

**Review by kruithofa — DISMISSED** [review]

**Comment by Thalley** [comment]
Since https://github.com/auto-pts/auto-pts/pull/1292 got merged in the auto-pts repo, I pulled in the commit to also change the service to be hexidecimal in this PR.

**Review by sjanc — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Review by kruithofa — APPROVED** [review]

**Pull request merged**